src/hooks/useDailyStatistics/utils.ts

```diff
@@ -39,16 +41,19 @@ export const countTotalTransactionsAndByCategory = (
               return key;
             })(),
           category: key,
-          quantityText: formatQuantityText(
-            totalQuantityInCategory,
-            findItemByCategory?.quantity.unit || {
-              type: "POSTFIX",
-              label: " qty",
-            }
-          ),
+
+          quantity: totalQuantityInCategory,
+          unit: findItemByCategory?.quantity.unit || {
+            type: "POSTFIX",
+            label: ` ${i18ntWithValidator(
+              "checkoutSuccessScreen",
+              "quantity"
+            )}`,
+          },
+
```

Comment:
Pardon me if I'm asking a question raised before, am also not 100% familiar with the context.

What the reason for replacing `formatQuantityText` here and letting the consumers handle it instead? I noticed the current code uses `quantityText` so the consumers of `useDailyStatistics` hook don't need to care about the specific details on how to format the quantity—they can just use the `quantityText` directly.